gcc-11.3.0/gcc/varasm.c

```diff
@@ -8193,9 +8194,10 @@ get_decimal (unsigned char c)
 	   8                   56                            8                */
 
 static void
-convert_ascii_hex_to_ascii_decimal (const char *in_array, char *out_array)
+convert_ascii_hex_to_ascii_decimal (const char *in_array, char *out_array,
+				    unsigned long in_array_half_len)
```

Comment:
Isn't the last argument the length of the hash ? why it is named half_len ?

Reply:
Because it is half of the length of the input array. The in_array has 40 (or 64) characters and we want 20 (or 32) characters in the out_array because we put that in the .note.gitbom section.